libs/ui/elements/src/lib/truncated-text/truncated-text.component.html

```diff
@@ -0,0 +1,23 @@
+<div class="flex items-center justify-between w-full gap-2">
+  <span
+    #textElement
+    class="truncate"
+    [class.cursor-pointer]="isTextTruncated"
+    [matTooltip]="isTextTruncated ? text : ''"
+  >
+    {{ text }}
+  </span>
+  <button
+    *ngIf="isTextTruncated"
+    [matTooltip]="text"
+    matTooltipPosition="above"
+    class="flex-shrink-0 border border-gray-300 px-2 py-1 rounded-lg w-11 h-9"
+  >
```

Comment:
This button is strange: it does not do anything on click. Looking at the mockup, I feel like the popover with the text should appear when clicking the button. Also, the popover looks like an actual block of text, and not just a `matTooltip`.

Reply:
And if the button shows the block of text on click, its tooltip should be something like "show all text".